metadata-utils/src/main/java/com/linkedin/metadata/authorization/PoliciesConfig.java

```diff
@@ -121,6 +121,9 @@ public class PoliciesConfig {
   public static final Privilege MANAGE_TAGS_PRIVILEGE =
       Privilege.of("MANAGE_TAGS", "Manage Tags", "Create and remove Tags.");
 
+  public static final Privilege VIEW_MANAGE_TAGS_PRIVILEGE =
+      Privilege.of("VIEW_MANAGE_TAGS", "View Manage Tags", "View the 'Manage Tags' page.");
+
```

Comment:
we should probably add this to `policies.json` so we can have this as default for admin roles and what not. probably editor too.

Reply:
yeah only thing is that manage tags is already there, and supercedes this privilege